code/auth-service/package.json

```diff
@@ -27,10 +27,11 @@
     "winston": "2.3.1"
   },
   "devDependencies": {
+    "@babel/cli": "^7.5.5",
+    "@babel/core": "^7.5.5",
+    "@babel/node": "^7.5.5",
+    "@babel/preset-env": "^7.5.5",
```

Comment:
I don't know if we need it here, but I also had to add these dependencies to support async/await syntax fir the infrastructure -api
@babel/plugin-transform-runtime": "^7.5.5",
@babel/runtime": "^7.5.5",

Reply:
Also, build is failing for this service